### DISCUSSION TITLE
Dumb questions/ Logs / Local gallery / Image generation

### DISCUSSION THREAD

**Discussion by Fraggle-Kong** [mudler/LocalAI#2206] [Q&A]
Dear all,

probably I got some things wrong.
I'm new to this for a couple of days now and was happy to be able to bring German language to my AI - I love it's rants, it's very entertaining. I use it with Nextcloud.
But now for the dumb questions.
I googled for hours, but either my questions are to dumb for results or I'm too dumb to find proper results.

First: how can I read OpenAI's logs? I installed it according to homepage with docker. Now, it's in the docker container, entertains me, but I don't know how to trouble shoot.
It my very first time with docker.

Isn't it supposed to serve a website to download models on http://myip:8080 ?
I can only use API (I have set up an API key).
Wenn I point my web browser to it, there's only a notice indicating lack of my authorisation key.
Is this intended?
What would I do to access local gallery?
Or has it just the API interface out of the box?

Third dumb question is about image generation.
Although I didn't fiddle around with image generation at all, Nextcloud just shows an Assistant error an Nextcloud's log shows:
`"[integration_openai] Warnung: API request error : Server error: `POST http://XXX.XXX.XXX.XXX:8080/v1/images/generations` resulted in a `500 Internal Server Error` response:
{"error":{"code":500,"message":"rpc error: code = Unavailable desc = error reading from server: EOF","type":""}}

	POST /ocs/v2.php/apps/assistant/api/v1/i/process_prompt
	von XXX.XXX.XXX.XXX von admin um 01.05.2024, 02:05:56"
`
My machine is capable of AVX2, but not AVX512.
I installed CPU version of OpenAI as it's in a docker container which is in a LXC Container on Proxmox - no GPU available.

Thanks for helping out a newbie!

Christian

**Comment by TiDjY35** [comment]
hello,

same error for me

my CPU 

**Flags:** 
fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon rep_good nopl xtopology cpuid tsc_known_freq pni pclmulqdq vmx ssse3 fma cx16 pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm cpuid_fault pti ssbd ibrs ibpb stibp tpr_shadow flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt arat vnmi umip md_clear arch_capabilities

and 

curl http://localhost:8080/v1/images/generations -H "Content-Type: application/json" -d '{
  "prompt": "floating hair, portrait, ((loli)), ((one girl)), cute face, hidden hands, asymmetrical bangs, beautiful detailed eyes, eye shadow, hair ornament, ribbons, bowties, buttons, pleated skirt, (((masterpiece))), ((best quality)), colorful|((part of the head)), ((((mutated hands and fingers)))), deformed, blurry, bad anatomy, disfigured, poorly drawn face, mutation, mutated, extra limb, ugly, poorly drawn hands, missing limb, blurry, floating limbs, disconnected limbs, malformed hands, blur, out of focus, long neck, long body, Octane renderer, lowres, bad anatomy, bad hands, text",
  "size": "256x256"
}'

{"error":{"code":500,"message":"rpc error: code = Unavailable desc = error reading from server: EOF","type":""}}




no problem other generation text

curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/json" -d '{
  "model": "llama-3.2-1b-instruct-q4_k_m.gguf",
  "messages": [{"role": "user", "content": "Say this is a test!"}],
  "temperature": 0.7
}'

{"created":1729445382,"object":"chat.completion","id":"e660dac5-0e66-4c58-8060-e85b1a5bf30f","model":"llama-3.2-1b-instruct-q4_k_m.gguf","choices":[{"index":0,"finish_reason":"stop","message":{"role":"assistant","content":"\n\nIt's just a text!"}}],"usage":{"prompt_tokens":17,"completion_tokens":8,"total_tokens":25}}
